packages/fluentui/docs/src/examples/components/Menu/Usage/MenuExampleWithSubmenuHover.shorthand.tsx

```diff
@@ -8,7 +8,6 @@ const items: MenuProps['items'] = [
     on: 'hover',
     menu: {
       items: [
-        { key: '1', content: 'item1' },
```

Comment:
Why this was removed?

Reply:
No reason, there was no need to have it there only

Reply:
Reverted